### ISSUE TITLE
Sec-WebSocket-Protocol header not present!

### ISSUE THREAD

**Issue by hasanr68** [hnakayam/webrtc-samples#1]
Hi there.

Just tried the "client" example - but get the following error:

Sec-WebSocket-Protocol header not present!

How to fix?

**Comment by hnakayam** [comment]
What web browser version do you use? I recommend using Chrome browser for testing.

There's one WebSocket specification of "Sec-WebSocket-Protocol" optional header but I never intended to use this. The other possibility is conflicting WebSocket port number of existing other service.

**Comment by hasanr68** [comment]
Hi again.
Tried with all 3 browsers. Really new to node.js and instead of ws:// is wrote wss:// - could that be the problem? Did edit client.js - but don't think the server picks up the change. (Unsure how to restart :D)

**Comment by hnakayam** [comment]
> ws:// is wrote wss:// 
Yes this is. This requests TLS/SSL connection to signaling server but signaling.js never accept TLS/SSL connection. 
This is not safe for the Internet but used here for simplicity.

**Comment by hnakayam** [comment]
Also, there's browser cache mechanism. In developer screen (F12) there exist html and js source code pain. you can "FORCE" refresh content form F12 developer screen, by right click the "Reload" button and "clear cache and hard refresh". This is only possible within F12 developer screen.

**Comment by hasanr68** [comment]
Hi again.
Thanks, I did the F12 trick - it picks up the wss -> ws change. Still doesn't connect to my server (not using signaling.js). It seems to want a certificate now. Do you intend to add Sec-Websocket-Protocol header?

**Comment by hnakayam** [comment]
Please note that WebRTC never defines standard SIGNALING PROTOCOL specification.
In this example one simple and unique SIGNALING PROTOCOL is implemented between signaling.js and client.js. You can implement your own SIGNALING PROTOCOL by modifying these sample implementations.
